client/src/App.vue

```diff
@@ -8,7 +8,7 @@
         crossorigin="anonymous"
       />
       <sidebar-menu
-        v-if="$auth.isAuthenticated"
+        v-if="this.$store.state.currentUser != {}"
```

Comment:
This prevents the user from seeing the navbar on the signup page, as well as other that are meant for unauth or new users.